BLSQ/iaso
# POLIO-666: Change FR translations

Some french translation on vaccine tab

## Self proof reading checklist

## Changes

- Change some french translation
- Replace fiole all over place in french translation with flacon

## How to test
- Change Version language into french version
- Edit a campaign
- Vaccine tab
- Check labels

## Print screen / video
![Iaso-Campagnes](https://user-images.githubusercontent.com/19631540/197979795-5e6f4699-346c-4a44-a59a-b24555f68e75.png)
